scala/src/com/google/idea/blaze/scala/run/producers/ScalaTestContextProvider.java

```diff
@@ -61,6 +64,14 @@ public RunConfigurationContext getTestContext(ConfigurationContext context) {
             testClass, TestSizeFinder.getTestSize(testClass));
     if (target == null) {
       return null;
+    } else {
+      try {
+        if(target.get(2, TimeUnit.SECONDS).kindString.equals("scala_junit_test")) {
```

Comment:
Please check if this .get or .kindString can return null, and if they can, check it.

Reply:
as I checked `.get` can only throw exceptions and `.kindString` is not marked @Nullable